### ISSUE TITLE
Improve local development set up of status dashboard

### ISSUE THREAD

**Issue by menghif** [Seneca-CDOT/telescope#2473]
It is currently possible to setup the new [status dashboard](https://api.telescope.cdot.systems/v1/status) for front-end development (with automatic reload) like this:
1. Add: `API_HOST=https://dev.telescope.cdot.systems` to [env.local](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/status/env.local)
1. Run: `npm run dev` from `src/api/status/`
2. Go to: http://localhost:1111/

This setup only works if we **delete** line 19 from `index.html`:
https://github.com/Seneca-CDOT/telescope/blob/80c3c0d6da46260017c2610765453ea68500da2f/src/api/status/public/index.html#L19

We need a way to make `npm run dev` work without having to delete line 19.
We should also update the docs to add the instructions above.

**Comment by humphd** [comment]
One way to fix this is to add an environment variable to `env.local` (maybe `PATH_PREFIX=/v1/status`) that when set causes use to use `/v1/status` instead of `/` for our static root here: https://github.com/Seneca-CDOT/telescope/blob/master/src/api/status/src/server.js#L33

```js
// For local dev, allow specifying a different path prefix (i.e., to mimic Traefik routing with our <base> tag)
const staticPathPrefix = process.env.PATH_PREFIX || '/'
service.router.use(staticPathPrefix, static(path.join(__dirname, '../public')));
```

**Comment by drew5494** [comment]
can I work on this issue?

**Comment by humphd** [comment]
Go for it @drew5494

**Comment by humphd** [comment]
Fixed by #2493. @drew5494, next time use `Fixes #...` in your PR description so it auto-closes.
